src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java

```diff
@@ -39,6 +39,10 @@
     // Yes/No field indicating if there are warnings to show
     public final String displayWarnings;
 
+    // Yes/No field indicating if the payments are processed
```

Comment:
indicating if the payment document control numbers are processed(payment processed is a different thing :) )

Reply:
updated